### DISCUSSION TITLE
SOLVED - Want to enable plugins - need help please

### DISCUSSION THREAD

**Discussion by Martin900t** [Ylianst/MeshCentral#4441] [Q&A] [answered]
I want to enable plugins to use https://github.com/ryanblenis/MeshCentral-ScriptTask . I've been trying to add the following to config.json but don't seem to be doing it correctly. Every time I restart the server I can't connect to the web page; it just keeps timing out.
I need to add:
"plugins": {
     "enabled": true
},

My current config.json
```
{
   "settings": {
      "cert": "domain.org", 
      "_minify": true, 
      "webrtc": false
   }, 
   "letsencrypt": {
      "email": "me@domain.org",
      "names": "domain.org",
      "rsaKeySize": 3072,
      "production": true
   },

   "domains": {
      
   }
}
```

Would someone mind putting the plugins:enabled in my json so I can get this working? I've spent hours on this and just can't get it figured out, Thanks.

**Comment by Martin900t** [comment, marked as answer]
Got it. I had to put a comma after "webrtc": false and I put both brackets on one line for "enabled": true .
Resulting config.json:

```
{
   "settings": {
      "cert": "domain.org", 
      "_minify": true, 
      "webrtc": false,
      "plugins": { "enabled": true }
   }, 
   "letsencrypt": {
      "email": "me@domain.org",
      "names": "domain.org",
      "rsaKeySize": 3072,
      "production": true
   },

   "domains": {
      
   }
}
```

**Comment by tecnicosya** [comment]
The plugins still working and supported?
I get this message on Meshcentral Status:
WARNING: Unrecognized configuration option "plugins".
And on webpanel say on my languaje: ADVERTENCIA: Opción de configuración desconocida "plugins".
And not show any option to install the plugin I want.

I try with add this line to settings:
"plugins": {"enabled": true}

**Comment by Martin900t** [comment]
Everything still seems fine here.

**Reply by tecnicosya** [reply]
Yes now plugins are working fine!

**Comment by HeyVern** [comment]
https://github.com/ryanblenis/MeshCentral-ScriptTask/issues/42

**Comment by itayiad** [comment]
I have a problem whereby plugins are failing to open schedules
![image](https://github.com/user-attachments/assets/2259dbcb-bb86-440a-b425-09fb4330d8fe)

**Reply by si458** [reply]
**plugins are not offically supported in meshcentral but provided as is**

you would need to speak to the plugin developer and find out why its not working
https://github.com/ryanblenis/MeshCentral-ScriptTask

if i get chance ill try adding the plugin here and find out whats wrong as it will be the plugin thats the issue
